calculator/calc.cpp

```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+using namespace std;
```

Comment:
stop the usage of `using namespace std;`

Reply:
resolved

Reply:
resolved